### ISSUE TITLE
orientation change during 'add venue' crashes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#40]
```
What steps will reproduce the problem?
1. Click "add venue"
2. During (i.e. immediately) or right at form load, change orientation.

Crash happens before app auto-populates address fields, not after.
```

Original issue reported on code.google.com by `amiantos@gmail.com` on 14 Aug 2009 at 5:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `amiantos@gmail.com` on 14 Aug 2009 at 5:53
- Changed title: **orientation change during 'add venue' crashes**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b2940c87d0.
```

Original comment by `jlapenna` on 17 Aug 2009 at 3:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b2940c87d0.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:47
